.pylintrc

```diff
@@ -2,11 +2,14 @@
 
 [MAIN]
 # Specify a score threshold to be exceeded before program exits with error.
-fail-under=9.7
+fail-under=9.8
 
 # Make sure public methods are documented.
 # See Also: https://github.com/PyCQA/pydocstyle/issues/309#issuecomment-1426642147
-fail-on=C0116
+# Also fail on unused imports.
+fail-on=
+    C0116,
```

Comment:
maybe write explicitly `missing-function-docstring` instead of cryptic C0116?

Reply:
These were just to try and get my local checks failing in the same way as you were seeing with flake8 but without making too many unrelated changes.  Let's take that up in #358